frontend/packages/kubevirt-plugin/package.json

```diff
@@ -9,6 +9,10 @@
     "@console/shared": "0.0.0-fixed",
     "kubevirt-web-ui-components": "~0.1.35"
   },
+  "scripts": {
+    "lint": "cd ../.. && eslint --ext .js,.jsx,.ts,.tsx --color packages/kubevirt-plugin",
```

Comment:
It would be great if we can reuse the script from `frontend/package.json` via sort of `yarn -cwd ../.. lint [TARGET]`.
Probably a new rule without `.` directory would need to be defined in the top-level `package.json`.

Reply:
good idea, created a #1813 for that so it is possible to lint even from top level